app/views/presentations/_form.html.erb

```diff
@@ -18,11 +18,9 @@
 
   <p>
     <%= f.label :event %><br>
-    <%= collection_select(:presentation, :event_id, Event.all, :id, :name, prompt: true) %>
+    <%= collection_select(:presentation, :event_id, Event.upcoming, :id, :name, prompt: true) %>
   </p>
 
-  <%= hidden_field_tag(:current_person, current_person.id) %>
```

Comment:
This line seems unrelated to this commit.
If this line is not needed, the history can be rebased so that it is never added.